app/models/monitoring_review.rb

```diff
@@ -0,0 +1,33 @@
+require "api_delegator"
+
+class MonitoringReview
+  attr_accessor :access_token, :id
+
+  def initialize(id, access_token)
+    @id = id
+    @access_token = access_token
+    @errors = []
+  end
+
+  def start_date
+    get_api_data_field :attributes, :reviewStartDate
+  end
+
+  def valid?
```

Comment:
[question] This is the part where I veered off from the active model validations and went offroading.  I'm not sure it's going to be successful in the long-run but I found it much more easy to understand what was happening. What do folks think?  Should I stick to how TTA is doing it (and therefore possibly pull some of the shared pieces into a concern) or will this be an okay way of doing things?

Reply:
I think this is fine for now. Once more error reporting is added (and the real API is nailed down) we might be able to converge a bit more. I think doing that now is premature.